projects/demo/src/modules/components/slider/examples/3/index.html

```diff
@@ -0,0 +1,5 @@
+<input tuiSlider value="1" max="4" size="m" [showTicks]="true" />
```

Comment:
I think ticks should not necessarily be tied to step, one can show ticks ever 2 steps or every 4. Lets keep previous API of showing a number of segments?

Reply:
I made input-property `[segments]` (as it was in previous API)